File: rb/utility/make_context.rb
# ChineseMedicineClinics SDK utility: make_context
require_relative '../core/context'
module ChineseMedicineClinicsUtilities
  MakeContext = ->(ctxmap, basectx) {
    ChineseMedicineClinicsContext.new(ctxmap, basectx)
  }
end

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// ChineseMedicineClinics SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class ChineseMedicineClinicsMakeContext
{
    public static function call(array $ctxmap, ?ChineseMedicineClinicsContext $basectx): ChineseMedicineClinicsContext
    {
        return new ChineseMedicineClinicsContext($ctxmap, $basectx);
    }
}
